A/Aeron/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Aeron"
+version = v"1.40.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/real-logic/aeron.git", "1cda80dbcd346ee0409fec5328956292614460df"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk update
+apk add openjdk11 hdrhistogram-c-dev libbsd-dev util-linux-dev
+cd $WORKSPACE/srcdir/aeron
+mkdir build && cd build
+CMAKE_FLAGS=(-DCMAKE_INSTALL_PREFIX=$prefix
+-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+-DCMAKE_BUILD_TYPE=Release
```

Comment:
I don't know what they had in mind in https://github.com/real-logic/aeron/blob/354608c4c3b12ecc363508da06fce2b52869db6d/CMakeLists.txt#L95-L97, but they very much use GNU extensions, so we very much need to enable extensions
```suggestion
-DCMAKE_BUILD_TYPE=Release
-DCMAKE_C_EXTENSIONS=ON
```

Reply:
Thanks! Great spot.